### PR TITLE
13092023 rahat

### PR DESCRIPTION
**Pull request by asadullahchowdhury**

### CODE REVIEW

**Review by rdhafiz — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/FrontController.php [hunk 1]
```diff
@@ -6,7 +6,18 @@
 
 class FrontController extends Controller
 {
-    public function home(){
+    public function home()
+    {
         return view('pages.home');
     }
+
+    public function homeBackground()
```
[the remark this answers is not in the document]
Comment (asadullahchowdhury) [reply app/Http/Controllers/FrontController.php:14]: Fixed

File: app/Http/Controllers/FrontController.php [hunk 2]
```diff
@@ -6,7 +6,18 @@
...
+    {
+        return view('pages.home-background');
+    }
+
+    public function homeTwoColumn()
```
[the remark this answers is not in the document]
Comment (asadullahchowdhury) [reply app/Http/Controllers/FrontController.php:19]: Fixed

File: resources/views/pages/home-background.blade.php
[the remark this answers is not in the document]
Comment (asadullahchowdhury) [reply resources/views/pages/home-background.blade.php:1]: Fixed

File: resources/views/pages/widgets/banner-background.blade.php
[the remark this answers is not in the document]
Comment (asadullahchowdhury) [reply resources/views/pages/widgets/banner-background.blade.php:1]: Fixed

File: routes/web.php [hunk 3]
```diff
@@ -14,4 +14,6 @@
 |
 */
 
-Route::get('/', [FrontController::class, 'home']);
+Route::get('/', [FrontController::class, 'home'])->name('home');
+Route::get('/homepage-with-background', [FrontController::class, 'homeBackground'])->name('home-background');
```
[the remark this answers is not in the document]
Comment (asadullahchowdhury) [reply routes/web.php:18]: Fixed

File: resources/views/layouts/components/header.blade.php
[the remark this answers is not in the document]
Comment (asadullahchowdhury) [reply resources/views/layouts/components/header.blade.php:1]: Fixed

**Review by rdhafiz — APPROVED** [review]

**Pull request merged**